config/jail.conf

```diff
@@ -767,3 +767,17 @@ port = 2222
 enabled  = false
 logpath  = /var/lib/portsentry/portsentry.history
 maxretry = 1
+
+[pass2allow-ftp]
+# this pass2allow example allows FTP traffic after successful HTTP authentication
+port         = ftp,ftp-data,ftps,ftps-data
+filter       = apache-pass
+# access log of the website with HTTP auth
+logpath      = %(apache_access_log)s
+# knocking URL needs to be customized per each deployment
+knocking_url = /secret-knocking-url
```

Comment:
ah -- actually now that you brought discussion back... I think I was wrong and mixed up filter and action definitions.  In action definitions we can pass those custom parameters into the action, but we never did that for filters... can we do smth like

```
filter = apache-pass[knocking_url=/secret-knocking-url]
```
please check... I don't think that defining it as a variable within jail would pass it in, but I could be wrong... neither not sure if above would work... if neither works -- just adjust above to state that they need e.g. ```filters.d/apache-pass.local``` defining that variable

sorry for the confusion and somewhat vague request here -- don't have time atm to dig in and investigate either we support those